tests/nosetests/pyanaconda_tests/module_timezone_test.py

```diff
@@ -120,3 +128,218 @@ def kickstart3_test(self):
         timezone Europe/Prague --ntpservers=ntp.cesnet.cz
         """
         self._test_kickstart(ks_in, ks_out)
+
```

Comment:
There should in a test for the `InstallWithTasks` method in `TimezoneInterfaceTestCase`.

Reply:
Added tests for default and some basic configuration in the "interface tests" commit.